### PR TITLE
Fix tests hang

### PR DESCRIPTION
**Pull request by Therzok**

### CODE REVIEW

File: main/tests/Ide.Tests/MonoDevelop.Ide.Editor/SkipCharSessionTests.cs [hunk 1]
```diff
@@ -40,25 +40,28 @@ public void TestBug58764 ()
 
 			var document = new Document (tww);
 
-			var editor = TextEditorFactory.CreateNewEditor (document);
-			editor.MimeType = "text/xml";
-			const string originalText = @"<?xml version=""1.0"" encoding=""utf-8"" ?>
+			using (var testCase = new TextEditorExtensionTestCase (document, content, tww, null, false)) {
```
Comment (KirillOsenkov) [inline main/tests/Ide.Tests/MonoDevelop.Ide.Editor/SkipCharSessionTests.cs]: >null [](start = 82, length = 4)

Consider using named arguments to name these literals.

**Comment by KirillOsenkov** [comment]
Thank you so much for working on this. :heart:

**Comment by Therzok** [comment]
md-addins PR successfully built!

**Comment by Therzok** [comment]
Still trying to reproduce, will possibly have something tomorrow

**Comment by Therzok** [comment]
Is this what I hope it is?

**Pull request merged**